File: Programming Basics/JavaScript/Conditional Statements Advanced/Labs/weekend-or-working-day.js
function solve(input) {
    const day = input[0];

    if (day === "Monday" || day === "Tuesday" || day === "Wednesday" || day === "Thursday" || day === "Friday") console.log("Working day");
    else if (day == "Saturday" || day == "Sunday") console.log("Weekend");
    else console.log("Error");
}

File: Programming Basics/C++/Conditional Statements Advanced/Labs/WeekendOrWorkingDay/WeekendOrWorkingDay.cpp
#include <iostream>
using namespace std;

int main()
{
	string day;
	cin >> day;

	if (day == "Monday" || day == "Tuesday" || day == "Wednesday" || day == "Thursday" || day == "Friday") cout << "Working day" << endl;
	else if (day == "Saturday" || day == "Sunday") cout << "Weekend" << endl;
	else cout << "Error" << endl;
}
